setup.py

```diff
@@ -1,48 +1,88 @@
 from __future__ import absolute_import, division, print_function
 
-from setuptools import setup, find_packages, Extension
+import glob
 import os
+import sys
+from setuptools import setup, find_packages, Extension
+from setuptools.command.build_ext import build_ext as _build_ext
 
-import numpy as np
+# Python 2 has a different name for builtins.
+try:
+    import builtins
+except ImportError:
```

Comment:
My understanding is that `builtins` doesn't work in Python 2. For example:
```Python 2.7.5 (default, Jun 11 2019, 12:19:05)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import builtins
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named builtins
```
The `six` library has a rename to use `_builtin_` for Python2 - see first line of table "Supported renames" at https://six.readthedocs.io/#module-six.moves but I thought it was better to avoid adding another dependency just for this use and DIY via catching ImportError.

If you change to Python 3 only in a later version, you can drop this check.

Reply:
I took this approach from numpy's `setup.py` such as https://github.com/numpy/numpy/blob/maintenance/1.16.x/setup.py#L33 and https://github.com/numpy/numpy/blob/maintenance/1.16.x/setup.py#L103

Reply:
@tjtg Cool... okay, I see the source of my confusion:
```shell
itwl@vld173:> python
Python 2.7.15 |Anaconda, Inc.| (default, Dec 14 2018, 19:04:19) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import builtins
>>> 
```
The `builtins` module must have been back-ported from Python3, as this works with Python `2.7.15`, but clearly not `2.7.5.:wink: